### PR TITLE
Maryhipp/trigger phrases

### PR DESCRIPTION
**Pull request by maryhipp**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Community Node Submission

## Have you discussed this change with the InvokeAI team?
- [x] Yes
- [ ] No, because:

## Description
* Update model manager to allow trigger phrase management as part of model metadata (subject to change with @psychedelicious 's schema rework)
* Adapt Embedding popover on prompts to work for trigger phrases, as well. Now only embeddings that are compatible with base model will be displayed alongside trigger phrases configured for that selected model. 

## QA Instructions, Screenshots, Recordings
Screenshot 2024-02-28 at 12 07 07 PM

Screenshot 2024-02-28 at 12 07 33 PM

## Added/updated tests?

- [ ] Yes
- [ ] No : _please replace this line with details on why tests
      have not been included_

### CODE REVIEW

File: invokeai/backend/model_manager/metadata/fetch/civitai.py [hunk 1]
```diff
@@ -164,6 +164,7 @@ def _from_model_json(self, model_json: Dict[str, Any], version_id: Optional[int]
                 AllowDerivatives=model_json["allowDerivatives"],
                 AllowDifferentLicense=model_json["allowDifferentLicense"],
             ),
+            trigger_phrases=version_json["trainedWords"],
```
Comment (psychedelicious) [inline invokeai/backend/model_manager/metadata/fetch/civitai.py:167]: This is already on the metadata object as `trained_words`

Comment (maryhipp) [reply invokeai/backend/model_manager/metadata/fetch/civitai.py:167]: Yeah I was trying to not use the civit key just to spite them

Comment (psychedelicious) [reply invokeai/backend/model_manager/metadata/fetch/civitai.py:167]: I mean the metadata schema already extracts it a few lines earlier

Comment (maryhipp) [reply invokeai/backend/model_manager/metadata/fetch/civitai.py:167]: Do you prefer `trained_words`? I don't feel strongly

Comment (lstein) [reply invokeai/backend/model_manager/metadata/fetch/civitai.py:167]: `trigger_phrases` works for me, and is more consistent with the terminology we use for textual inversion embeddings.

**Review by lstein — COMMENTED** [review]
The MM backend changes look fine to me.

**Pull request closed without merging**